### ISSUE TITLE
Instructions out of date in building template

### ISSUE THREAD

**Issue by WizardOfAus** [alexa/skill-sample-nodejs-adventure-game#78]
For: https://github.com/alexa/skill-sample-nodejs-adventure-game/blob/master/instructions/1-voice-user-interface.md

Some of the instructions such as the step to follow after sign in is outdated as there seems to be a change in the UI and this may be confusing for some.

**Comment by ozrohepworth** [comment]
Yes, the instructions are out of date. I was in Stage 2: Lambda Function and after adding the trigger and saving, I went to the next step, i.e. **Upload the zip file lambda/custom/gamebook.zip** 
I tried: https://github.com/alexa/skill-sample-nodejs-adventure-game/tree/master/lambda/custom
but it does not have the file mentioned in the instructions. 

Could you please let us know if this will be resolved? 
Thanks.
